package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cerebral-cli",
-  "version": "0.0.13",
+  "version": "0.0.15",
```

Comment:
0.0.15 is current version in master/ seems you forgot to use latest master when you start branch